src/yaws_config.erl

```diff
@@ -821,6 +822,15 @@ fload(FD, globals, GC, C, Cs, Lno, Chars) ->
                      {error, ?F("Expect integer at line ~w", [Lno])}
              end;
 
+        ["soap_workers_recycle", '=', Bool] ->
+             if (Bool == "true") ->
```

Comment:
Man ifs are ugly in Erlang... `if (Bool == "true") -> [...]; true -> [...]` . wtf. Anyway, it works.